src/equals.js

```diff
@@ -4,7 +4,14 @@
 // var abs = Math.abs;
 
 module.exports = function equals(p1, p2) {
```

Comment:
But this one I have mostly kicked out trying to minimize function calls. However it didn't give much gain

Reply:
Simple operations like this won't have much performance gains in the grand scheme of things.

However, dropping `for loops` helps a lot!

Reply:
yeah this was a bit of a long shot but I figured if it was a micro-optimisation that was being called a squillion times it might help, see https://jsperf.com/andcomparitor/1 (although the results are depenedent on what browser you use)